### PR TITLE
chore: update cloud provider and csi image versions in VHD

### PR DESCRIPTION
**Pull request by haofan-ms**
**Reason for Change**:

update cloud provider and csi image versions in VHD

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [ ] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: vhd/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -49,8 +49,13 @@ function Get-ContainerImages {
...
                 "mcr.microsoft.com/oss/kubernetes/pause:1.4.1",
                 "mcr.microsoft.com/oss/kubernetes/pause:3.4.1",
-                "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.0.1-alpha.1-windows-1809-amd64",
-                "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v1.2.1-alpha.1-windows-1809-amd64")
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager:v1.0.8",
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager:v1.1.4",
+                "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.10.0",
+                "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.4.0",
+                "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.5.0",
+                "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:0.0.12",
```
Comment (haofan-ms) [inline vhd/packer/configure-windows-vhd.ps1:57]: I am not sure if this secrets-store provider image is needed on the Windows VHD for agent node. Please advise @jackfrancis

File: vhd/packer/configure-windows-vhd.ps1 [hunk 2]
```diff
@@ -49,8 +49,13 @@ function Get-ContainerImages {
                 "mcr.microsoft.com/windows/nanoserver:1809",
                 "mcr.microsoft.com/oss/kubernetes/pause:1.4.1",
                 "mcr.microsoft.com/oss/kubernetes/pause:3.4.1",
-                "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.0.1-alpha.1-windows-1809-amd64",
-                "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v1.2.1-alpha.1-windows-1809-amd64")
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager:v1.0.8",
```
Comment (jackfrancis) [inline vhd/packer/configure-windows-vhd.ps1:52]: I think we don't need this, as the `cloud-controller-manager` pod will never be scheduled onto a Windows node.

Is there a Windows `cloud-node-manager` container image @marosset @jsturtevant ?

File: vhd/packer/configure-windows-vhd.ps1 (hunk 1 above)
Comment (jackfrancis) [reply vhd/packer/configure-windows-vhd.ps1:57]: Are you confident that these container images are multi-arch? (linux and windows)

File: vhd/packer/configure-windows-vhd.ps1 (hunk 2 above)
Comment (haofan-ms) [reply vhd/packer/configure-windows-vhd.ps1:52]: Sorry I was copying the wrong image url, the one we need is cloud-node-manager, let me update

Comment (jsturtevant) [reply vhd/packer/configure-windows-vhd.ps1:52]: yes for out of tree: https://github.com/kubernetes-sigs/cloud-provider-azure/blob/960a8ee5ae26dc1be4e2dbf53f4f865b28c72e6d/examples/out-of-tree/cloud-node-manager.yaml#L105

File: vhd/packer/configure-windows-vhd.ps1 (hunk 1 above)
Comment (haofan-ms) [reply vhd/packer/configure-windows-vhd.ps1]: I verified the cloud-node-manager and csi images are working fine on Windows nodes. The secrets-store images are not needed on Windows as the secrets-store pods are only on linux nodes.

**Review by marosset — APPROVED** [review]

File: vhd/packer/configure-windows-vhd.ps1 [hunk 3]
```diff
@@ -49,8 +49,11 @@ function Get-ContainerImages {
                 "mcr.microsoft.com/windows/nanoserver:1809",
                 "mcr.microsoft.com/oss/kubernetes/pause:1.4.1",
                 "mcr.microsoft.com/oss/kubernetes/pause:3.4.1",
-                "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.0.1-alpha.1-windows-1809-amd64",
-                "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v1.2.1-alpha.1-windows-1809-amd64")
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.0.8",
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.1.4",
+                "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.10.0",
```
Comment (jackfrancis) [inline vhd/packer/configure-windows-vhd.ps1:54]: Are these 3 CSI images Windows container images?

Comment (marosset) [reply vhd/packer/configure-windows-vhd.ps1:54]: Yes - https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/deploy/csi-azuredisk-node-windows.yaml

Comment (marosset) [reply vhd/packer/configure-windows-vhd.ps1:54]: The liveness probe and node-driver-registrar containers are shared by all CSI plugin.

Comment (haofan-ms) [reply vhd/packer/configure-windows-vhd.ps1:54]: In AKSe, we have different image reference for Linux and Windows pointing to the same image (https://github.com/Azure/aks-engine/blob/master/pkg/api/k8s_versions.go#L39-L42), and I have checked pods on windows nodes, and they are using these images.

**Review by marosset — APPROVED** [review]
/lgtm

**Comment by jackfrancis** [comment]
Thank you @marosset!

**Pull request merged**